File: src/main/java/com/akagiyui/drive/model/request/EnableConfigRequest.java
package com.akagiyui.drive.model.request;

import jakarta.validation.constraints.NotNull;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 开关配置 请求
 * @author AkagiYui
 */
@Data
@NoArgsConstructor
public class EnableConfigRequest {
    @NotNull
    private Boolean enabled;
}

File: app/src/main/kotlin/com/akagiyui/drive/model/request/EnableConfigRequest.kt
package com.akagiyui.drive.model.request

import jakarta.validation.constraints.NotNull

/**
 * 开关配置 请求
 * @author AkagiYui
 */
class EnableConfigRequest {
    /**
     * 是否启用
     */
    @NotNull
    var enabled: Boolean = false
}
